src/overlays/actors/ovl_Bg_Toki_Swd/z_bg_toki_swd.c

```diff
@@ -93,19 +93,19 @@ void BgTokiSwd_Destroy(Actor* thisx, PlayState* play) {
 }
 
 void func_808BAF40(BgTokiSwd* this, PlayState* play) {
-    if (!GET_EVENTCHKINF(EVENTCHKINF_WATCHED_SHEIK_AFTER_MASTER_SWORD_CS) && !IS_CUTSCENE_LAYER &&
+    if (!GET_EVENTCHKINF(EVENTCHKINF_REVEAL_MASTER_SWORD_CS) && !IS_CUTSCENE_LAYER &&
```

Comment:
Sorry last change, I should have caught this earlier.
I think the flag can be `EVENTCHKINF_REVEALED_MASTER_SWORD`
it gets set to indicate that it has already been revealed and that the cs shouldnt play again.